src/target/ext/extended_mode.rs

```diff
@@ -95,6 +95,9 @@ pub trait ExtendedMode: Target {
     /// In all-stop mode, all threads in the attached process are stopped; in
     /// non-stop mode, it may be attached without being stopped (if that is
     /// supported by the target).
+    ///
+    /// After this method resolves, GDB will immediately request a stop reason.
+    /// It is up to the implementation to ensure the stop reason is accurate.
```

Comment:
this doesn't apply anymore, right?

Reply:
I admit I wasn't entirely sure about that comment, but yes, that does not apply now so I have removed it.